IOSExercises/AppDelegate.m

```diff
@@ -0,0 +1,105 @@
+//
+//  AppDelegate.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 10/28/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+
+@interface AppDelegate ()
```

Comment:
Remove unnecessary code blocks (this is fine to be removed if no declared internal properties)